cloud-director/v31.0.0/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+resources:
+- release.yaml
+
+replacements:
+- source:
+    group: release.giantswarm.io
+    kind: Release
+    fieldPath: metadata.name
+    options:
+      delimiter: "-"
+      index: 2
```

Comment:
Probably not easily doable, but there was a comment about why we use `index: 2` here.